### PR TITLE
fix(components/subheaderbar): design adjustments

### PR DESCRIPTION
**Pull request by ncomont**
**What is the problem this PR is trying to solve?**

Subheader does not follow the actual guidelines.

**What is the chosen solution to this problem?**

Align it.

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Actions/Action/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import Action, { wrapOnClick } from './Action.component';
+import Action, { wrapOnClick, ACTION_TYPES } from './Action.component';
 
-export { wrapOnClick };
+export { wrapOnClick, ACTION_TYPES };
```
Comment (jmfrancois) [inline packages/components/src/Actions/Action/index.js]: why do you export ACTION_TYPES ?

File: packages/components/src/EditableText/EditableText.scss [hunk 2]
```diff
@@ -46,6 +46,8 @@ $tc-circle-editable-text-size: 16px !default;
 		padding-left: $padding-small;
 		padding-right: $padding-small;
 		flex: 0 0 auto;
+		line-height: inherit;
+		min-height: inherit;
```
Comment (frassinier) [inline packages/components/src/EditableText/EditableText.scss]: Could you try with IE 11, please?
https://caniuse.com/#feat=css-unset-value
Just to be sure

File: packages/components/src/Actions/Action/index.js (hunk 1 above)
Comment (jmfrancois) [reply packages/components/src/Actions/Action/index.js]: I prefer you to attach it to the Action component so you can write
```
Action.DISPLAY_MODE_FILE
```
etc.

File: packages/components/src/SubHeaderBar/SubHeaderBar.scss [hunk 3]
```diff
@@ -14,15 +14,17 @@ $tc-svg-icon-size: $padding-large !default;
 	width: 100%;
 
 	button {
```
Comment (jmfrancois) [inline packages/components/src/SubHeaderBar/SubHeaderBar.scss]: nothing seems attached to button, Is this possible to remove it ?

**Pull request merged**